apps/src/templates/levelSummary/summary.module.scss

```diff
@@ -98,23 +105,39 @@ label.sectionSelector {
     column-gap: 48px;
     padding-top: 24px;
     padding-bottom: 12px;
+
+    .studentResponseBlock {
+      margin-bottom: 16px;
+    }
+
+    .studentName {
+      min-height: 24px;
+
+      p {
+        margin: 0;
+        font-size: 14px;
+        font-weight: bold;
+        padding-left: 12px;
+      }
+    }
   }
 
   .studentAnswer {
-    padding: 12px;
-    margin-bottom: 16px;
     // TODO: Update to use #E6F6F8 once the phase-2 brand colors are in.
-    background-color: $lightest_cyan;
```

Comment:
Move/remove this TODO comment as appropriate?

Reply:
Yeah, good point, I believe we are updating this color to purple anyway.  Will remove.